mujinvisioncontrollerclient/visioncontrollerclient.py

```diff
@@ -38,38 +38,41 @@ class VisionControllerClient(object):
     _checkpreemptfn = None # called periodically when in a loop
     
     _subscriber = None # an instance of ZmqSubscriber, used for subscribing to the state
-    
+
+    _deprecated = None # used to mark arguments as deprecated (set argument default value to this)
+
     def __init__(self, hostname='127.0.0.1', commandport=7004, ctx=None, checkpreemptfn=None, reconnectionTimeout=40, callerid=None):
         # type: (str, int, Optional[zmq.Context], Optional[Callable], float, Optional[str]) -> None
```

Comment:
To make sure, when there is a default value like `reconnectionTimeout=40` then it is not `Optional`. That is only in the case when the value can be `None`

Reply:
Yes.

It clashed a bit with how we set optional in docstrings. I dont like it myself. In the future I would like to get rid of type and optional annotation in docstrings at all actually. Since now it will become redundant. But it is better to do in a separate MR since it may be more controversial.